webapp/config.ru

```diff
@@ -0,0 +1,20 @@
+ROOT = File.expand_path(File.dirname(__FILE__))
+$LOAD_PATH.unshift File.join(ROOT, 'lib')
+Dir.glob('lib/**').each{ |d| $LOAD_PATH.unshift(File.join(ROOT, d)) }
+
+
+env = (ENV['RACK_ENV'] || :development).to_sym
```

Comment:
I don't know why, I tend to not use fetch, but is a good example here where is doing a good job. Thanks for noticing.